docs/content/doc/advanced/hacking-on-gitea.en-us.md

```diff
@@ -32,24 +32,24 @@ necessary. To be able to use these you must have the `"$GOPATH"/bin` directory
 on the executable path. If you don't add the go bin directory to the
 executable path you will have to manage this yourself.
 
-**Note 2**: Go version 1.9 or higher is required, however it is important
+**Note 2**: Go version 1.9 or higher is required; however, it is important
 to note that our continuous integration will check that the formatting of the
 source code is not changed by `gofmt` using `make fmt-check`. Unfortunately,
-the results of `gofmt` can differ by the version of `go` it is therefore
+the results of `gofmt` can differ by the version of `go`. It is therefore
 recommended to install the version of go that our continuous integration is
-running. At the time of writing this is Go version 1.11, however this can be
+running. At the time of writing this is Go version 1.11; however, this can be
```

Comment:
We're now on go 1.12.

`modules/setting/setting.go` is an example where gofmt has changed its output between 1.11 and 1.12 (at least it appears so)